### DISCUSSION TITLE
Get Calendar Updates

### DISCUSSION THREAD

**Discussion by YazeedFares** [ical4j/ical4j#781] [Q&A]
Hi, 

I am new to calendar integration functionality and I was wondering if there is a way to send a message from the calendar entry if someone modified it.

To clarify, we have a web app where people can create events, the events can be sent to participants as meeting invitations, the problem is there is no way to prevent editing the calendar entry on the calendar app, so is there a way to send a notification from the calendar app to the web app to identify the changes?

**Comment by benfortuna** [comment]
Unfortunately that is entirely up to the calendar app the event was imported into. I think the only way you might be notified is by setting the organiser property to an email address that can handle proposal of a new event time, but that would still rely on the user to send updates when they change the event.
